README.md

```diff
@@ -58,6 +58,12 @@ you set above with the --volume-snapshot-locations flag
 velero backup create my-backup --include-namespaces=my-namespace --snapshot-volumes --volume-snapshot-locations vsl-vsphere
 ```
 
+If attempting to backup the entire cluster please exclude the upload and download crds by using the --exclude-resources flag
+
+```bash
+velero backup create my-backup --snapshot-volumes --volume-snapshot-locations vsl-vsphere --exclude-resources uploads.veleroplugin.io,downloads.veleroplugin.io,uploads.veleroplugin.io,downloads.veleroplugin.io
+```
```

Comment:
"uploads.veleroplugin.io,downloads.veleroplugin.io" are repeated in this example.

Also that's the old API group.  See new API group here: 
https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/main/pkg/apis/datamover/v1alpha1/register.go#L34

Reply:
I meant to specify the singular and plural versions of the resources. Fixed it now. For 1.0.1 and 1.0.2, the names weren't changed to datamover yet: https://github.com/vmware-tanzu/velero-plugin-for-vsphere/tree/1.0.2/pkg/generated/crds/manifests